### ISSUE TITLE
Feature Request: Example project with blazor UI framework?

### ISSUE THREAD

**Issue by gismofx** [webview/webview_csharp#4]
Is there any possibility to create a blazor example, or get the blazor sample project to run in webview?

I'm not expert in this area, but is there some overlap with this similar project? [WebWindow](https://github.com/SteveSandersonMS/WebWindow)

**Comment by geaz** [comment]
Yes, webview core and webwindow try to solve the same requirement. 

A blazor example would be possible. Will add it later maybe.

**Comment by gismofx** [comment]
Thanks for clarifying and for the consideration of a Blazor example.

**Comment by congzhangzh** [comment]
@gismofx how about use webview to open the asp.net in process(worker thread) directly?  that's more reliable as even Microsoft asp.net server side blazor implement in asp.net abi changed, you do not need change your application, just recompile

**Comment by congzhangzh** [comment]
@gismofx give https://github.com/congzhangzh/desktoploveblazorweb a try?
